controllers/Frontend.php

```diff
@@ -10,52 +12,66 @@ public function home_cv()
     {
         $model_form = new Form();
         
-        $page="frontend/home_cv";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $formulaire = $model_form->form_page_home_cv();
+        $myView = new View('home_cv');
+        $myView->render();
     }
 
     public function legalnotice()
     {
-        $page="frontend/legalnotice";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('legalnotice');
+        $myView->render();
     }
     
     public function home()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
-
-        $page="frontend/home";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $manager = new ArticleManager();
+        $posts = $manager->get_posts_blog();
+        
+        $myView = new View('home');
+        $myView->render(array('posts' => $posts));
     }
 
     public function blog()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
+        $manager = new ArticleManager();
+        $posts = $manager->get_posts_blog_all();
         
-        $page="frontend/blog";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('blog');
+        $myView->render(array('posts' => $posts));
     }
 
     public function post()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
-                
-        $page="frontend/post";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        if(isset($_GET['id']))
+        {
+            $id = $_GET['id'];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a non-sanitized input variable: $_GET['id']](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)